securedrop_client/gui/widgets.py

```diff
@@ -1037,6 +1037,74 @@ def __init__(self):
         self.update()
 
 
+class LoginErrorBar(QWidget):
+    """
+    A bar widget for displaying messages about login errors to the user.
+    """
+
+    CSS = '''
+    QWidget {
+        background-color: #f22b5d;
```

Comment:
should we update this to #ce0083 (what's in zeplin)?

Reply:
@ninavizz - is this error bar color for the login window supposed to be different than the error bar color in the main window? I think both look great! Just want to double-check.

Reply:
ok, i just confirmed with nina that the design is intended to have two different error colors because of the different surrounding-ui contexts.